lib/membrane_live/webinars/webinar.ex

```diff
@@ -0,0 +1,43 @@
+defmodule MembraneLive.Webinars.Webinar do
+  @moduledoc """
+  Struct for webinar
+  """
+  use Ecto.Schema
+  import Ecto.Changeset
+
+  @type t :: %__MODULE__{
+          description: String.t(),
+          moderator_link: String.t(),
+          presenters: list,
+          start_date: NaiveDateTime.t(),
+          title: String.t(),
+          viewer_link: String.t()
+        }
```

Comment:
We could also think about generating viewer_link as `/event/<primary_key>`, this way we will not have to store it in DB as a separate field. But this way it is also good.

Example tutorial for setting it up https://elixircasts.io/uuid-primary-key-with-ecto.

Reply:
I've addressed this issue in [this comment](https://github.com/membraneframework-labs/membrane_live/pull/4#issuecomment-1189887305)